docs/en/setup/service-agent/java-agent/agent-optional-plugins/Logger-plugin.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+
+`logger-plugin` can store the logs generated by the program during the call, such as the content of the error log, into the span log.  Through the configuration file, you can control the log source (log4j2, logback, log4j), package name, and level. Please pay attention that  the original `log4j.xml`(or other log framework config files) has high priority than the `logconfig.properties`.
+
+# Configuration file
+
+By default, the configuration file is in `apache-skywalking-apm-bin/agent/config/logger-plugin/logconfig.properties`.Of course, **If the file does not exist, the default configurations are as following,**
+
+```properties
+log4j.packages=*
+log4j.level=error
+log4j2.packages=*
+log4j2.level=error
+logback.packages=*
+logback.level=error
+```
+
+The meaning of the above configuration is as follows:
+
+1. SkyWalking opened the adaptor(bridge) between tracing kernel and log frameworks, including `log4j`, `log4j2`, `logback`.
+2. Only collect logs at the `error` level, others would be ignored, including `trace`, `debug`, `info`, `warn`.
+3. Wouldn't filter the logs by the package name.
+
+# Property description
+
+## packages
+
+**package attribute**: Specify the package name of the log that needs log conversion.  **The default value is `*` , and it will match all packages.**
```

Comment:
```suggestion
**package attribute**: Specify the package name of the log that needs log conversion. **The default value is `*`, and it will match all packages.**
```